composer.lock

```diff
@@ -1571,16 +1571,16 @@
         },
         {
             "name": "phpactor/class-to-file",
-            "version": "dev-master",
+            "version": "0.5.1",
```

Comment:
I don't know what happened here. That could be a downgrade.

Reply:
yeah also not sure, but it's a downgrade indeed